drivers/hmis/app/models/hmis/hud/enrollment.rb

```diff
@@ -302,34 +294,20 @@ def save_new_enrollment!
   end
 
   def save_in_progress!
-    # "WIP" Enrollments have a null ProjectID column, so that they don't
-    # get pulled into reporting. The Project database ID is stored in the hmis_wips table.
-    saved_project_id = project.id
-
+    # "WIP" Enrollments have a null ProjectID column, so that they aren't included in reporting.
     self.project_id = nil
```

Comment:
Should we raise an error here if `project_pk` is unset? It's not expected, but we should guard against it here so we don't lose both the project links